PlayerOne/index.html

```diff
@@ -38,8 +38,8 @@ <h5>Player Two's Previous Guesses</h5>
         <script src="./index.js"></script>
         <script src="./canvas.js"></script>
         <script src="playerTwoGuesses.js"></script>
-        <script src="submitWordToGuess.js"></script>
+        <!-- <script src="submitWordToGuess.js"></script> -->
```

Comment:
This one is not being created anywhere, therefore the network is giving an error